mail_tracking/models/mail_tracking_email.py

```diff
@@ -92,6 +92,15 @@ class MailTrackingEmail(models.Model):
         string="Tracking events", comodel_name='mail.tracking.event',
         inverse_name='tracking_email_id', readonly=True)
 
+    @api.multi
+    def write(self, vals):
+        if 'state' in vals and vals['state'] in \
+                ['error', 'rejected', 'spam', 'bounced', 'soft-bounced']:
```

Comment:
As said above, use a method if possible. Anyways:

```suggestion
                {'error', 'rejected', 'spam', 'bounced', 'soft-bounced'}:
```